sunpy/timeseries/tests/test_timeseriesbase.py

```diff
@@ -413,6 +414,16 @@ def test_ts_index(generic_ts):
 def test_ts_sort_index(generic_ts):
     assert generic_ts.sort_index().data.equals(generic_ts.data.sort_index())
 
+#_validate_units
+
+#_validate_meta
+
+@skip_wcsaxes

```

Comment:
I don't think this needs `skip_wcsaxes` that is for tests that require `wcsaxes` i.e. are doing coordinate aware image plotting.